src/Tribe/Ticket_Object.php

```diff
@@ -660,5 +698,16 @@ public function get_event() {
 
 			return null;
 		}
+
+		/**
+		 *
+		 */
+		public function show_description() {
+				$show = ( metadata_exists( 'post', $this->ID, $this->show_description_key ) ) ?
+				get_post_meta( $this->ID, $this->show_description_key, true ) :
+				true;
+
+				return apply_filters( 'tribe_show_ticket_description', $show );
```

Comment:
Also: event-tickets actions/filters have a lot of legacy names. However, we do have a [naming standard](http://moderntribe.github.io/products-engineering/plugins/) that says we should prefix action/filter names with `tribe_tickets_`.